pkg/ccl/backupccl/restore_online.go

```diff
@@ -76,12 +163,7 @@ func sendAddRemoteSSTs(
 		return 0, 0, errors.AssertionFailedf("online restore can only restore data from a full backup")
 	}
 
-	restoreSpanEntriesCh := make(chan execinfrapb.RestoreSpanEntry, 1)
-
-	grp := ctxgroup.WithContext(ctx)
-	grp.GoCtx(func(ctx context.Context) error {
-		return genSpan(ctx, restoreSpanEntriesCh)
-	})
+	const targetSpanSize = 440 << 20
```

Comment:
I hesitate to use "logical span" since we're moving towards using that for backup spans nee manifest_files. These are ranges, and range sizes are always logical, so I think this should just be `targetRangeSize`.